---
title: LOCALDB_ERROR_INTERNAL_ERROR | Microsoft Docs
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.technology: performance
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: 809420b875ebf20b45efb6b297cb198cfed8422b
ms.sourcegitcommit: f7fced330b64d6616aeb8766747295807c92dd41
ms.translationtype: MT
ms.contentlocale: pt-BR
ms.lasthandoff: 04/23/2019
ms.locfileid: "62990395"
---
# <a name="localdberrorinternalerror"></a>LOCALDB_ERROR_INTERNAL_ERROR
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Detalhes  
  
|||  
|-|-|  
|Nome do produto|SQL Server|  
|ID do evento|264|  
|Origem do evento|Tempo de execução de banco de dados local do SQL Server 12.0|  
|Componente|API do tempo de execução de banco de dados local|  
|Texto da mensagem|Ocorreu um erro inesperado dentro de uma chamada de método API de instância de banco de dados local. Consulte o log do Aplicativo do Windows para obter detalhes sobre o erro.|  
  
## <a name="explanation"></a>Explicação  
 Erro inesperado.  
  
## <a name="user-action"></a>Ação do usuário  
 Consulte o log de eventos para obter detalhes.  
  
  
